gateway-js/src/__tests__/gateway/testUtils.ts

```diff
@@ -66,8 +66,16 @@ export async function startSubgraphsAndGateway(
     serviceList.push({ name: serviceDef.name, url });
   }
 
+  const mockLogger = {
```

Comment:
Out of curiosity, is that to avoid the warnings about `serviceList` being deprecated during tests?

Not strongly opposed to this, but I am a little uneasy to blanket silence all warn and error logs. Basically, I'd worry about us making a stupid mistake that results in spamming some warning/error and it making it to a release. I'd agree the status quo, logging stuff during tests, is not perfect, but it at least give us a better chance to catch such error.

Reply:
Removed. I actually don't remember why this ended up here, possibly stray mid-development stuff that I left. Definitely agree we don't want to silence completely.

We could eventually adopt this and create expectations on the logger which would allow us to both quiet the logs and be specific with our log expectations but I don't think it belongs in this PR.